oap-server/server-library/library-elasticsearch-client/src/main/java/org/apache/skywalking/library/elasticsearch/requests/IndexRequest.java

```diff
@@ -29,5 +31,7 @@ public final class IndexRequest {
     private final String index;
     private final String type;
     private final String id;
+    @Builder.Default
+    private final Optional<String> routing = Optional.empty();
```

Comment:
This should be the routing value, rather than the routing column name.

Please follow https://github.com/apache/skywalking/pull/10282#discussion_r1071853153, and update this and add comments.